src/schema/v2/credit_card.ts

```diff
@@ -118,6 +118,7 @@ export const {
   edgeType: CreditCardEdge,
 } = connectionDefinitions({
   nodeType: CreditCardType,
+  nonNullable: true,
```

Comment:
This one seems to have that null pattern too: https://github.com/artsy/metaphysics/blob/42a1752e7a46457fb5cac1ad6c02c7723c6660c6/src/schema/v2/me/credit_cards.ts#L13